src/main.ts

```diff
@@ -1138,22 +1136,30 @@ export default class RemotelySavePlugin extends Plugin {
       return;
     }
 
-    this.syncOnRemoteIntervalID = window.setInterval(async () => {
+    const syncOnRemote = async () => {
       if (this.syncStatus !== "idle") {
         return;
       }
 
       const metadataMtime = await this.getMetadataMtime();
 
       if (metadataMtime === undefined) {
-        return;
+        return false;
       }
 
       if (metadataMtime !== this.settings.lastSynced) {
         log.debug("Sync on Remote ran | Remote Metadata:", metadataMtime + ", Last Synced:", this.settings.lastSynced);
         this.syncRun("auto");
+        return true;
       }
-    }, this.settings.syncOnRemoteChangesAfterMilliseconds);
+    };
+
+    if (Platform.isMobileApp) {
```

Comment:
I am confused, why does this only occur for mobile (and maybe could you explain why this is happening here)?

Reply:
So the toggle function will only run once when Sync On Remote is enabled or plugin loads. This then sets an interval for the remote checking. 

I needed a way to check if it was the first iteration of the interval. So before the interval is set to the syncOnRemote function it calls the function and it returns whether remote changes were found or not. It then prints the notices based on that.

I think it can work well for desktop as well but just implemented what was originally planned. Please remove the if statement if you feel it would be better.

Reply:
That makes sense. I forgot the context for that half, making it easier to see status at launch *on mobile* specifically (because Desktop has the status bar as an option). Thank you!